caddyhttp/browse/setup.go

```diff
@@ -426,7 +426,9 @@ footer {
 								<span class="name">{{html .Name}}</span>
 							</a>
 						</td>
-						{{- if .IsDir}}
+						{{- if .IsSymlink }}
+						<td data-order="-1">symbolic link</td>
```

Comment:
I can update the graphics. How about we take this out (and leave the template unchanged) for now and I will see about getting icons with little arrows or something to indicate a symbolic link?